File: src/test/java/com/csl9015/StudentTest.java
package com.csl9015;

import com.csl9015.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestScoreTest() {
        Student student = new Student("Willy", 80, 95);
        Assertions.assertEquals(95, student.highest());
    }

    @Test
    public void averageTest() {
        Student student = new Student("Kent", 98, 90);
        Assertions.assertEquals((98 + 90) / 2, student.getAverage());
    }
}

File: src/test/kotlin/com/csl9015/StudentTest.kt
package com.csl9015.kotlin


import com.csl9015.student.Studentk
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestTest() {
        val studentk = Studentk("Emly", 95, 88)
        Assertions.assertEquals(95, studentk.highest())
    }

    @Test
    fun averageTest() {
        val studentk = Studentk("Emly", 98, 88)
        Assertions.assertEquals((98 + 88) / 2, studentk.getAverage())
    }

    @Test
    fun gradingTest() {
        val studentk = Studentk("Emlt", 98, 88)
        Assertions.assertEquals('A', studentk.grading())
    }
}